CalibCalorimetry/HcalPlugins/python/Hcal_Conditions_forGlobalTag_cff.py

```diff
@@ -35,7 +35,7 @@
         qieSlope      = cms.vdouble(0.912,0.917,0.922,0.923),
         mcShape       = cms.int32(125),
         recoShape     = cms.int32(105),
-        photoelectronsToAnalog = cms.double(0.0),
+        photoelectronsToAnalog = cms.double(0.3305),
         darkCurrent   = cms.vdouble(0.0),
```

Comment:
Previously this value was only stored in Python (https://github.com/cms-sw/cmssw/blob/CMSSW_9_0_X/SimCalorimetry/HcalSimProducers/python/hcalSimParameters_cfi.py). Now it will be used in Method 2 reconstruction as well as digi simulation, so it will be taken from the database as noted in my above comment and the PR description. Once this PR and the conditions updates are accepted, the Python config can be cleaned up to reduce duplication.